content/rest-api/filter.md

```diff
@@ -469,6 +469,7 @@ As seen previously, the attribute type determines which set of operators is avai
 | Allowed operators | Allowed value type |
 | ---------- | ---------- |
 | =, != | boolean |
+| EMPTY, NOT EMPTY | no value  (only on Serenity version)|
```

Comment:
Actually, the `NOT EMPTY` filter was already available in previous versions (although not documented)
The `EMPTY` filter is available as of v5.0

Reply:
We will update the documentation for the 5.0 by replacing all the `only on Serenity version` by `available since the 5.0`